TUTORIAL.md

```diff
@@ -432,7 +455,7 @@ type Config struct {
 	// checking that all pods are ready by default with 8 minutes timeout
 	//	&client.ReadyCheckData{
 	//		ReadinessProbeCheckSelector: "",
-	//		Timeout:                     8 * time.Minute,
+	//		Timeout:                     15 * time.Minute,
```

Comment:
This is the only line change that matters in this doc, the rest were auto formatter fixes